app/controllers/facility_reservations_controller.rb

```diff
@@ -46,7 +46,7 @@ def edit
 
     raise ActiveRecord::RecordNotFound unless @reservation == Reservation.find(params[:id])
     set_windows
-    unless @reservation.can_edit? || @reservation.can_edit_actuals?
+    unless @reservation.admin_editable? || @reservation.can_edit_actuals?
```

Comment:
Can we extract this `||` check into a method since it's also used down below? I would see if there's a way we can phrase the positivity/negativity of the name to flip the two  `unless`es into `if`s.